morph/Visual.h

```diff
@@ -255,6 +258,30 @@ namespace morph {
         // A callback friendly wrapper
         static void set_context (morph::Visual<glver>* _v) { _v->setContext(); };
 
+        //! Lock the context mutext to prevent locking across multiple. Then sets the context.
+        void lockContext() { 
+            context_mutex.lock(); 
+            setContext();
+        }
+        
+        //! Attempt to lock the context mutext returns if locking was successful. 
+        //! If success sets the context.
+        bool tryLockContext() { 
+          if(context_mutex.try_lock()){
```

Comment:
4 spaces indentation, { on newline after function (but not after if). this-> specifiers to indicate class membership of attributes/functions

Reply:
Changed all the above.